fetch/out/model_RMSEs.csv

```diff
@@ -0,0 +1,116 @@
+exper_n,exper_id,model_type,rmse
```

Comment:
You don't want to commit anything that is considered a "build artifact" of the pipeline, meaning anyone who runs the pipeline can recreate locally. Revisit the second paragraph of the "Activity" section [here](https://github.com/Kfaunce/ds-pipelines-targets-1/pull/4) to see details about this. Essentially, you should edit your `.gitignore` file so that none of the `.csv`, `.txt`, or `.png` files are committed to the repo.